### PR TITLE
Complying with new plugin distribution

### PR DESCRIPTION
**Pull request by git-afsantos**
Providing a fix for #62.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
-pyflwor_ext==1.2
-rospkg==1.1.4
-bonsai_code==0.4.5
-setuptools==20.7.0
-PyYAML==3.13
+pyflwor_ext>=1.2
```
Comment (gavanderhoorn) [inline requirements.txt]: I realise it was already there before, but isn't the correct name `pyflwor-ext` instead?

Or does `pip` check underscores and dashes automatically?

Comment (git-afsantos) [reply requirements.txt]: The original file was automatically generated, I think. And it seems [from this SO answer](https://stackoverflow.com/a/19131777) that `pip` replaces the underscore automatically.

**Comment by gavanderhoorn** [comment]
Btw @git-afsantos I must say that you're support with haros has been absolutely excellent. :+1: :100:  and :1st_place_medal:.

Seriously.

**Comment by git-afsantos** [comment]
Thanks! :blush: 

When I find the issue to be small enough to fix, I would rather get it done asap, so both the user and I can move along with our work. Otherwise it is going to fall into a *to do*, and who knows when that will be.

**Pull request merged**